Zilon.Core/Zilon.GlobeObserver/Program.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Linq;
+
+using Microsoft.Extensions.DependencyInjection;
+
+using Zilon.Core.Persons;
+using Zilon.Core.World;
+
+namespace Zilon.GlobeObserver
+{
+    class Program
+    {
+        static async System.Threading.Tasks.Task Main()
+        {
+            var serviceContainer = new ServiceCollection();
+            var startUp = new StartUp();
+            startUp.RegisterServices(serviceContainer);
+
+            serviceContainer.AddSingleton<IGlobeInitializer, GlobeInitializer>();
+            serviceContainer.AddSingleton<IGlobeExpander>(provider => (BiomeInitializer)provider.GetRequiredService<IBiomeInitializer>());
+            serviceContainer.AddSingleton<IGlobeTransitionHandler, GlobeTransitionHandler>();
+            serviceContainer.AddSingleton<IPersonInitializer, AutoPersonInitializer>();
+
+            var serviceProvider = serviceContainer.BuildServiceProvider();
+
+            // Create globe
+            var globeInitializer = serviceProvider.GetRequiredService<IGlobeInitializer>();
+            var globe = await globeInitializer.CreateGlobeAsync("intro");
+
+            // Iterate globe
+            var globeIterationCounter = 0L;
+            do
+            {
+                Console.WriteLine("Iteratin count:");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=6105716)